executor/show.go

```diff
@@ -166,11 +166,28 @@ func (e *ShowExec) fetchShowEngines() error {
 	return nil
 }
 
+// let information_schema be the first database
+func replaceInformationFirst(dbs []string) {
+	if strings.EqualFold(dbs[0], "INFORMATION_SCHEMA") {
+		return
+	}
+
+	first := dbs[0]
+	for i := 1; i < len(dbs); i++ {
+		if strings.EqualFold(dbs[i], "INFORMATION_SCHEMA") {
+			dbs[i] = first
+			dbs[0] = "INFORMATION_SCHEMA"
```

Comment:
how about:
```go
copy(dbs[1:i], dbs[0:i-1])
dbs[0] = "INFORMATION_SCHEMA"
```

Thus we have moved the "INFORMATION_SCHEMA" to the first element and the remained elements are sorted in the origin ascending order.

Reply:
good, Thanks!

Reply:
Another question:
how to run just one file or one test case for the function,  like TestShowVisibility in the show_test.go to replace make dev for all test files.

Reply:
Oh, I think it should be `copy(dbs[1:i+1], dbs[0:i])`